runtime/tests/interpreter/interpreter_test.go

```diff
@@ -5146,6 +5146,38 @@ func TestInterpretDictionaryKeyTypes(t *testing.T) {
 	}
 }
 
+func TestPathToString(t *testing.T) {
```

Comment:
In addition to the interpreter test, could you please also add a similar checker test to `runtime/tests/checker/path_test.go`?

Reply:
Certainly! Added. This is the first test I've added for the check so let me know if/how to improve it. 🙂